saleor/order/utils.py

```diff
@@ -338,3 +343,18 @@ def get_voucher_discount_for_order(order: Order) -> Money:
 
 def match_orders_with_new_user(user: User) -> None:
     Order.objects.confirmed().filter(user_email=user.email, user=None).update(user=user)
+
+
+def _get_unique_pdf_file_name():
+    while True:
+        pdf_path = f"{settings.MEDIA_ROOT}/{token_urlsafe(32)}.pdf"
+        if not os.path.isfile(pdf_path):
+            return pdf_path
```

Comment:
What about people that don't want local media storage, i.e. scaling? I would definitely not want this part making system calls onto my application